statsmodels/datasets/tests/test_utils.py

```diff
@@ -18,8 +18,9 @@ def test_get_rdataset():
         duncan = get_rdataset("Duncan", "car", cache=cur_dir)
         assert_(duncan.from_cache)
 
-# temporarily disabled because some test servers don't have internet
-def t_est_webuse():
+internet_available = check_internet()
+@dec.skipif(internet_available is False)
```

Comment:
Hmm, makes sense. I somehow have been sticking to my old habit of being explicit.
I am going to fix this.